docs/guides/authentication.md

```diff
@@ -0,0 +1,67 @@
+# Authentication
+
+marimo provides a simple way to add token/password protection to your marimo server. Given that authentication is a complex topic, marimo does not provide a built-in authentication/authorization system, but instead makes it easy to add your own through ASGI middleware.
+
+## Enabling Basic Authentication
+
+Authentication is disabled by default, however this may change in the future. To enable authentication, you must pass `--token` to your `marimo edit/run/new` command from the Terminal. This will use a randomly generated token in `Edit mode` and a deterministic token in `Run mode` (based on the code of the notebook).
```

Comment:
@mscolnick Update this sentence to indicate that auth is enabled by default in `edit/tutorial/new` ?